ui.apps/src/main/content/jcr_root/apps/venia/components/commerce/productlist/v1/productlist/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="nt:unstructured"
+    jcr:title="Product List"
+    sling:resourceType="cq/gui/components/authoring/dialog">
+    <content
+        jcr:primaryType="nt:unstructured"
+        sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns"
+        margin="{Boolean}false">
+        <items jcr:primaryType="nt:unstructured">
+            <column
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="granite/ui/components/coral/foundation/container">
+                <items jcr:primaryType="nt:unstructured">
+                    <pageSize
```

Comment:
Why was this added to cq_dialog? We already have cq_design_dialog, I think the property config should go there, no need for an additional dialog.

Reply:
it was discussed in forum and concluded that we want to keep `pageSize` authorable , thus cq_dialog was added, should we add this back in design dialog?

Reply:
Yes `pageSize` should be authorable but having two dialogs for very similar author settings does not make any sense. In the design dialog it is authorable as well. I have no strong preference here, as long as we only have one dialog.

Reply:
`pageSize` is only in cq_dialog, its not included in cq_design_dialog.
as discussed we are keeping in cq_dialog so it is authorable at page level.